recipes/vtk/build.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+if [ `uname` == Linux ]; then
+    CC=${PREFIX}/bin/gcc
+    CXX=${PREFIX}/bin/g++
```

Comment:
Not sure I understand how they were incompatible. We regularly build things with the system compiler that link against Python. Could we please try doing so?